### PR TITLE
Added application name to magic

### PR DESCRIPTION
**Pull request by patrick-nicholson**
Addresses #23

### CODE REVIEW

File: spylon_kernel/init_spark_magic.py [hunk 1]
```diff
@@ -37,8 +39,9 @@ def cell_init_spark(self):
 
         globals_dict = self.env
         exec(self.code, globals_dict)
+        application_name = globals_dict.get('application_name', None)
```
Comment (parente) [inline spylon_kernel/init_spark_magic.py]: I'm looking at a diff quickly, but I'm guessing you could just put "ScalaMetaKernel" here in place of the None and remove the check later. Also, it should probably be "spylon-kernel" by default.

Comment (patrick-nicholson) [inline spylon_kernel/init_spark_magic.py]: How about a global `DEFAULT_APPLICATION_NAME` in `scala_interpreter.py` that can be used in both files? I just used `"ScalaMetaKernel"` as it's the `init_spark_session` default.

Comment (patrick-nicholson) [inline spylon_kernel/init_spark_magic.py]: Since I instantiate it as `None`, `get` is never actually going to return the default. Removing it for clarity.

**Comment by parente** [comment]
🍰

**Pull request merged**